newIDE/app/src/ObjectGroupsList/index.js

```diff
@@ -589,6 +592,13 @@ const ObjectGroupsList = React.forwardRef<Props, ObjectGroupsListInterface>(
                         canMoveSelectionToItem={canMoveSelectionTo}
                         reactDndType={groupWithContextReactDndType}
                         initiallyOpenedNodeIds={initiallyOpenedNodeIds}
+                        renderHiddenElements={
```

Comment:
This could have been named `overscanMoreHiddenItems`. I think it would be fair to have this at 6 by default (this won't kill a desktop device by any mean), up to 20 like now when this is enabled

Reply:
I wonder if this could always be 20 actually. Won't kill a desktop device, and still ok on mobile.